mozci/push.py

```diff
@@ -533,6 +535,39 @@ def scheduled_duration(self):
 
         return int(duration / 3600)
 
+    def is_group_running(self, group):
+        """Checks if the provided group is still running on this push.
+
+        Returns:
+            bool: True, if the group is still running.
+                  False, if the group is completed.
+        """
+        running_tasks = [
+            task
+            for task in self.tasks
+            if task.state not in ["completed", "failed", "exception"]
```

Comment:
This makes me think that we should update `completed_cached_tasks` to also consider `failed` and `exception`, we were only caching `completed` :)

Reply:
Would you like me to make the update in this PR or elsewhere?

Reply:
You could do it in this PR, defining this list in the defs.py file